Tests/test_utils.py

```diff
@@ -57,6 +58,70 @@ def run_command(command, is_silenced=True, exit_on_error=True):
     return output
 
 
+def filter_packagify_changes(modified_files, added_files, removed_files, tag='master'):
```

Comment:
release_notes.py is importing functions from this file.(test_utils.py) so I think it should import this one also and avoid the copy-code